queries/injections.scm

```diff
@@ -18,3 +18,15 @@
 
 ((comment) @injection.content
  (#set! injection.language "jsdoc"))
+
+; Parse Ember/Glimmer/Handlebars/HTMLBars/etc. template literals
+; e.g.: await render(hbs`<SomeComponent />`)
+(call_expression
+  function: ((identifier) @_name
+             (#eq? @_name "hbs"))
+  arguments: ((template_string) @glimmer
+              (#offset! @glimmer 0 1 0 -1)))
+
+; Ember Unified <template> syntax
+; e.g.: <template><SomeComponent @arg={{double @value}} /></template>
+((glimmer_template) @glimmer)
```

Comment:
I got it!

![image](https://user-images.githubusercontent.com/199018/177221858-f88660ef-9f85-46a7-8e1a-e7282169009c.png)

I figured out how to test this locally!

working great!
(and now further highlight token can be created in tree-sitter-glimmer)